src/lib/db.js

```diff
@@ -1,80 +1,105 @@
-export const data = [
-  {
-    id: 0,
-    question: 'What type of a language is HTML?',
-    answer: 'Markup Language',
-    isBookmarked: true,
-    tags: ['HTML', 'general'],
-  },
-  {
-    id: 1,
-    question: 'Inside which HTML element do we put the JavaScript?',
-    answer: '<script>',
-    isBookmarked: true,
-    tags: ['HTML', 'Javascript', 'general'],
-  },
-  {
-    id: 2,
-    question: 'Who maintains the CSS specifications?',
-    answer: 'World Wide Web Consortium.',
-    isBookmarked: true,
-    tags: ['CSS'],
-  },
-  {
-    id: 3,
-    question: 'Who designed the Linux OS?',
-    answer: 'Linus Torvalds',
-    isBookmarked: true,
-    tags: ['Linux'],
-  },
-  {
-    id: 4,
-    question:
-      'Which of the following key combinations allows to terminate the current process in Linux shell?',
-    answer: 'Ctrl + C',
-    isBookmarked: false,
-    tags: ['Linux', 'Shell', 'Shortcuts'],
-  },
-  {
-    id: 5,
-    question: 'What is the correct way to write a JavaScript array?',
-    answer: 'const colors = ["red", "green", "blue"]',
-    isBookmarked: true,
-    tags: ['Javascript', 'Array', 'general'],
-  },
-  {
-    id: 6,
-    question: 'What should be the first tag in any HTML Document?',
-    answer: '<!doctype html>',
-    isBookmarked: false,
-    tags: ['HTML', 'general'],
-  },
-  {
-    id: 7,
-    question: 'How can you add a comment in a JavaScript?',
-    answer: '//This is a comment',
-    isBookmarked: false,
-    tags: ['Javascript', 'general'],
-  },
-  {
-    id: 8,
-    question: 'What tag is used to display a picture in a HTML page?',
-    answer: 'Img',
-    isBookmarked: false,
-    tags: ['HTML', 'HTML-Tag', 'general', 'picture'],
-  },
-  {
-    id: 9,
-    question: 'HTML are web pages read and rendered by?',
-    answer: 'Web Browser',
-    isBookmarked: false,
-    tags: ['HTML', 'general'],
-  },
-  {
-    id: 10,
-    question: 'Who created Git?',
-    answer: 'Linus Torvalds',
-    isBookmarked: true,
-    tags: ['Git'],
-  },
-]
+export function pushSection(sectionObject) {
+  console.log(sectionObject)
+  let questionCards
+  const storageQuestions = JSON.parse(localStorage.getItem('questionCards'))
+  if (storageQuestions === null) {
+    questionCards = []
+  } else {
+    questionCards = storageQuestions
+  }
+
+  questionCards.push(sectionObject)
+  localStorage.setItem('questionCards', JSON.stringify(questionCards))
+}
+
+export function getSection() {
+  let questionCards
+  const storageQuestions = JSON.parse(localStorage.getItem('questionCards'))
+  if (storageQuestions === null) {
+    questionCards = []
+  } else {
+    questionCards = storageQuestions
+  }
+  return questionCards
+}
+
+// export const data = [
```

Comment:
No worries, you can also remove this from the code to leave it cleaner. It won't go anyways as git will keep it there for sure.